sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -127,10 +128,18 @@
   @Override
   public Span.Builder addLink(Link link) {
     Utils.checkNotNull(link, "link");
+    totalNumberOfLinksAdded++;
+    // don't bother doing anything with any links beyond the max.
+    // todo: generate a metric for dropped links
```

Comment:
I think this makes more sense to be an issue for all dropped things (attributes, links, events).

Reply:
definitely

Reply:
so suggestion is to remove the todo and create an issue to discuss and make a decision on what to do with the dropped things.

Reply:
ok, will do.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/766

Reply:
todo removed.